nixpkgs_review/report.py

```diff
@@ -109,7 +109,10 @@ def markdown(self, pr: Optional[int]) -> str:
         cmd = "nixpkgs-review"
         if pr is not None:
             cmd += f" pr {pr}"
-        msg = f"Result of `{cmd}` [1](https://github.com/Mic92/nixpkgs-review)\n"
+
+        system = subprocess.check_output(["nix-instantiate", "--eval", "--json", "--expr", "builtins.currentSystem"], text=True).strip('"')
+
+        msg = f"Result of `{cmd}` [1](https://github.com/Mic92/nixpkgs-review) run on {system}\n"
```

Comment:
```suggestion
        msg = f"Result of `{cmd}` run on {system} [1](https://github.com/Mic92/nixpkgs-review)\n"
```

We should probably move this to the end.